endpoints/openrtb2/auction.go

```diff
@@ -126,6 +126,9 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		labels.Source = pbsmetrics.DemandApp
 		labels.RType = pbsmetrics.ReqTypeORTB2App
 		labels.PubID = effectivePubID(req.App.Publisher)
+		if _, found := deps.cfg.BlacklistedAppMap[labels.PubID]; found {
+			return
```

Comment:
We might want to write some sort of error back on `w` at this point before simply returning at this point. The only other places we return are in the case of a severe error, so I can kind of understand not writing something out there. Since this could be a common occurrence, I worry that simply returning will cause connections to hang until a timeout. Probably worth at least testing if the return will close the connection immediately or hang the connection.

Reply:
Corrected Hans. Updated PR now writes to `w`